src/index.js

```diff
@@ -45,11 +45,11 @@ const render = (Component, store, error) => {
   } else {
     ReactDOM.render(
       <Provider store={store}>
-        <ConnectedRouter history={history}>
+        <BrowserRouter basename='hg'>
```

Comment:
```suggestion
        <BrowserRouter basename={window.HGAC_BASEPATH || '.'}>
```

We should also make sure `window.HGAC_BASEPATH` is a properly formatted path, i.e., has a leading but no trailing slash.